Formula/j/jdtls.rb

```diff
@@ -3,9 +3,9 @@ class Jdtls < Formula
 
   desc "Java language specific implementation of the Language Server Protocol"
   homepage "https://github.com/eclipse-jdtls/eclipse.jdt.ls"
-  url "https://www.eclipse.org/downloads/download.php?file=/jdtls/milestones/1.30.1/jdt-language-server-1.30.1-202312071447.tar.gz"
-  version "1.30.1"
-  sha256 "4c005ede9df73e60cfb8f611373808c9121286d3adbfb745384cced9f19b2de3"
+  url "https://www.eclipse.org/downloads/download.php?file=/jdtls/milestones/1.31.0/jdt-language-server-1.31.0-202312071447.tar.gz"
```

Comment:
url is wrong

Reply:
https://www.eclipse.org/downloads/download.php?file=/jdtls/milestones/1.31.0/jdt-language-server-1.31.0-202401111522.tar.gz

Reply:
yeah, bump command should not be used for this case. (we should probably have bum account checking the page link though)